packages/playwright-test/src/runner.ts

```diff
@@ -619,11 +618,15 @@ function filterByFocusedLine(suite: Suite, focusedTestFileLines: TestFileFilter[
   if (!filterWithLine)
     return;
 
-  const testFileLineMatches = (testFileName: string, testLine: number, testColumn: number) => focusedTestFileLines.some(filter => {
-    const lineColumnOk = (filter.line === testLine || filter.line === null) && (filter.column === testColumn || filter.column === null);
+  const testFileLineMatches = (testFileName: string, testLine: number, testColumn: number) => focusedTestFileLines.some(({ re, exact, line, column }) => {
+    const lineColumnOk = (line === testLine || line === null) && (column === testColumn || column === null);
     if (!lineColumnOk)
       return false;
-    return createFileMatcherFromFilters([filter])(testFileName);
```

Comment:
Why did this change? It was introduced in a recent bug fix that unified the matchers, unless there is a need for that let's revert this part of the PR.

Reply:
sorry, I messed up trying to rebase things